plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -201,14 +202,15 @@
 
   $: updateIssueStatusId(object, currentProject)
   $: updateAssigneeId(object, currentProject)
+  $: updateComponentId(object, currentProject)
   $: canSave = getTitle(object.title ?? '').length > 0 && object.status !== undefined && kind !== undefined
 
   $: empty = {
     assignee: assignee ?? currentProject?.defaultAssignee,
     status: status ?? currentProject?.defaultIssueStatus,
     parentIssue: parentIssue?._id,
     description: '<p></p>',
-    component: component ?? $activeComponent ?? null,
+    component: component ?? $activeComponent ?? currentProject?.defaultComponent ?? null,
```

Comment:
Probable also we need to take component from parent issue as well if paren issue is defined.